app/views/membership.html

```diff
@@ -134,8 +134,12 @@ <h3>
                       </ui-select-match>
                       <ui-select-choices
                         repeat="role as role in filteredRoles | filter: excludeExistingRoles(subject.roles) | filter: { metadata: { name: $select.search } } track by (role | uid)">
-                        <div ng-bind-html="role.metadata.name | highlight: $select.search"></div>
-                        <div ng-if="role | annotation : 'description'">
+                        <div
+                          ng-bind-html="role.metadata.name | highlight: $select.search"
+                          title="{{role.metadata.name}}"></div>
```

Comment:
Can we specify this once on the parent node instead of once on each child?

Reply:
Tried that, but you can't put it directly on the `ui-select-choices`.  So I could instead add a new `div`, but then we have a `div` just for a `title`, which isn't awesome either.